qiskit/quantum_info/states/random.py

```diff
@@ -117,18 +117,18 @@ def _ginibre_matrix(nrow, ncol, seed):
     Args:
         nrow (int): number of rows in output matrix.
         ncol (int): number of columns in output matrix.
-        seed(int or RandomState): RandomState for rng.
+        seed(int or default_rng): default_rng for rng.
```

Comment:
```suggestion
        seed(int or np.random.Generator): default_rng for rng.
```

can you do a grep for `"or default_rng"` and change all of them

Reply:
This is done.

Reply:
@nonhermitian I still see "or default_rng" appearing in some places.